package/Dockerfile

```diff
@@ -61,6 +65,12 @@ VOLUME /var/log/auditlog
 RUN mkdir -p /usr/share/rancher/ui && \
     cd /usr/share/rancher/ui && \
     curl -sL https://releases.rancher.com/ui/${CATTLE_UI_VERSION}.tar.gz | tar xvzf - --strip-components=1 && \
+    cd assets && \
+    mkdir -p rancher-ui-driver-linode && \
```

Comment:
use `mkdir -p assets/rancher-ui-driver-linode && \`
then `cd assets/rancher-ui-driver-linode` just in case assets isn't there